src/stravalib/tests/integration/test_client.py

```diff
@@ -156,7 +155,7 @@ def test_get_activity_laps(mock_strava_api, client):
     )
     laps = list(client.get_activity_laps(42))
     assert len(laps) == 2
-    assert laps[0].distance == meters(1000)
+    assert laps[0].distance == 1000
```

Comment:
These returns will now be "plain" floats or ints (which will later get properties or other accessors to retrieve the value as a quantity).

Reply:
sounds good. what is the plan to add such properties / what will that look like in the model?